frontend/packages/operator-lifecycle-manager/src/components/operand/DEPRECATED_operand-form.tsx

```diff
@@ -716,7 +718,10 @@ export const DEPRECATED_CreateOperandForm: React.FC<OperandFormProps> = ({
           ? immutableFormData.setIn(['metadata', 'namespace'], match.params.ns).toJS()
           : immutableFormData.toJS(),
       )
-        .then(() => history.push(next))
+        .then((res) => {
+          postFormCallback(res);
+          return history.push(next);
+        })
```

Comment:
Any reason to not wait for the form actions to complete first?
```suggestion
        .then((res) => postFormCallback(res))
        .then(() => history.push(next))
```